apps/api/src/app/feature-flags/use-cases/get-feature-flag/get-feature-flag.use-case.ts

```diff
@@ -33,6 +33,17 @@ export class GetFeatureFlag {
     return await this.execute(command);
   }
 
+  async isTopicNotificationEnabled(featureFlagCommand: FeatureFlagCommand): Promise<boolean> {
+    const value = process.env.FF_IS_TOPIC_NOTIFICATION_ENABLED;
+    const fallbackValue = true; // It is a permanent feature now
+    const defaultValue = this.prepareBooleanStringFeatureFlag(value, fallbackValue);
+    const key = FeatureFlagsKeysEnum.IS_TOPIC_NOTIFICATION_ENABLED;
+
+    const command = this.buildCommand(featureFlagCommand, key, defaultValue);
+
+    return await this.execute(command);
+  }
```

Comment:
Every feature flag can have its dedicated method to check the value or go to the fallback.